apps/web/modules/users/views/users-public-view.tsx

```diff
@@ -58,6 +58,13 @@ export function UserPage(props: InferGetServerSidePropsType<typeof getServerSide
 
   const isEventListEmpty = eventTypes.length === 0;
   const isOrg = !!user?.profile?.organization;
+
+  const allowSEOIndexing = isOrg
+    ? isOrgSEOIndexable
+      ? profile.allowSEOIndexing
+      : false
+    : profile.allowSEOIndexing;
```

Comment:
The intention was to give priority to org (to which user belongs to) settings over user settings when SEOIndexing is disabled in Org.
And considering the security/privacy concern that if Org want to disable SEOIndexing, the user page will be disabled (regardless of user setting).

If only Org setting has to be enforced regardless of setting, then can be updated accordingly.

Reply:
Currently,
_What if the User has seoIndexing disabled but Organization has it enabled. Do we want to enforce Organization setting there?_ - Since Org setting is enabled, no concern from org for enabling SEO Indexing, so User setting is taken 

_What if seoIndexing is disabled in Organization but enabled in user. Do we still want to enforce the org setting to user?_ - Here since it is disabling, Org Setting is prioritized and taken regardless of user setting.

Let me know, if need to update.

Reply:
Yeah, we are discussing internally. I'll update here

Reply:
may i know if there is any update.

Reply:
@hariombalhara , Thanks for sharing,
What i infer is - UI should reflect the changes we are doing internally.
1. OrgSeo is disabled, User Seo is also set to false and switch (UI) is disabled.
2. OrgSeo is enabled, User Seo switch is enabled.

Reply:
@hariombalhara , implemented this in UI .
if Org.AllowSeoIndexing is false, then user's AllowSeoIndexing Switch is disabled and shown false.
if Org.AllowSeoIndexing is true, then user's AllowSeoIndexing Switch is enabled and user's choice is shown.

Reply:
@hariombalhara , to add - `Org.AllowSeoIndexing` is false by default.

Reply:
By having it false by default, it effectively disables seo indexing for all by orgs by default. If some users of some organization have seo indexing enabled, it would disable it for them.

This is why we discussed about "No explicit value" which is when org.allowSeoIndexing is null


But I am thinking that orgs would normally not want to have SEO indexing enabled and it might be better to have it disabled automatically for all org members, unless organization allows it first.